main.cc

```diff
@@ -2503,4 +2534,160 @@ void soup_up(int player) {
 		player.morl = 8;
 	}
 }
+
+void change_club(int new_club_idx, int new_player) {
+	struct gamea::manager &manager = gamea.manager[new_player];
```

Comment:
Just `player`. (It's at most a two-player hot-seat game, and you're either player0 or player1.)